### PR TITLE
Fix ACN configuration

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Fixes the ACN configuration for the agent.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/agents/oracle/aea-config.yaml [hunk 1]
```diff
@@ -89,17 +89,17 @@ public_id: valory/p2p_libp2p_client:0.1.0
...
-  save_path: .certs/acn_cosmos_11000.txt
-is_abstract: false
+- identifier: ${str:acn}
+  ledger_id: ${str:ethereum}
+  message_format: ${str:'{public_key}'}
+  not_after: ${str:'2023-01-01'}
+  not_before: ${str:'2022-01-01'}
+  public_key: ${str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
+  save_path: ${str:.certs/acn_cosmos_11000.txt}
+is_abstract: ${bool:false}
```
Comment (Adamantios) [inline packages/valory/agents/oracle/aea-config.yaml:102]: Cannot successfully run `autonomy hash all` after this change. Getting the following error:

```
ValueError: Configuration of component (connection, valory/p2p_libp2p_client:0.1.0) is not valid. sequence item 1: expected str instance, int found
```

Comment (angrybayblade) [reply packages/valory/agents/oracle/aea-config.yaml:102]: `is_abstract` is not overridable and AFAIC the only thing overridable in the `cert_request` object is a public key

**Review by 0xArdi — APPROVED** [review]

File: packages/valory/agents/oracle/aea-config.yaml [hunk 2]
```diff
@@ -89,17 +89,17 @@ public_id: valory/p2p_libp2p_client:0.1.0
...
 - identifier: acn
   ledger_id: ethereum
   message_format: '{public_key}'
-  not_after: '2023-01-01'
-  not_before: '2022-01-01'
-  public_key: ${P2P_PUBLIC_KEY:str:03c74dbfbe7bbc1b42429f78778017a3cd7eaf9d59d1634c9505a3f7c1a9350e71}
-  save_path: .certs/acn_cosmos_11000.txt
-is_abstract: false
+  not_after: '2022-01-01'
+  not_before: '2021-01-01'
```
Comment (0xArdi) [inline packages/valory/agents/oracle/aea-config.yaml:99]: not sure that is OK.

Comment (Adamantios) [reply packages/valory/agents/oracle/aea-config.yaml]: fixed.

**Review by 0xArdi — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**